### ISSUE TITLE
Testing 2.0 on Jungle - get schedule doesn't work

### ISSUE THREAD

**Issue by jrosich** [EOSIO/eos#8045]
I have set up a fresh new node with 2.0.0-rc1. cleos get schedule throws the following error:

    Producer      Producer key
    ============= ==================
error 2019-10-09T11:01:15.831 cleos     main.cpp:3997                 main                 ] Failed with error: Key Not Found (6)
Key block_signing_key

**Comment by heifner** [comment]
What endpoint did you use?

```
programs/cleos/cleos -u https://jungle2.cryptolions.io:443 get schedule
active schedule version 276
    Producer      Producer key
    ============= ==================
    alohaeostest  EOS6CHhStoknSrJiQuKMezu54QsYUsWQNnooNiBfDUFzRQzfE9ruq
    atticlabjbpn  EOS5hVMcN6UVWtrNCxdp5HJwsz4USULmdfNA22UDyjRNdprXEiAP6
    batinthedark  EOS6dwoM8XGMQn49LokUcLiony7JDkbHrsFDvh5svLvPDkXtvM7oR
    bighornsheep  EOS5xfwWr4UumKm4PqUGnyCrFWYo6j5cLioNGg5yf4GgcTp2WcYxf
    bigpolarbear  EOS6oZi9WjXUcLionUtSiKRa4iwCW5cT6oTzoWZdENXq1p2pq53Nv
    blocksmithio  EOS6vgrS4BXid4QfZqDM95xGz1LsvdJLoMrAPx3CFBUQGQ1Qsrs73
    clevermonkey  EOS5mp5wmRyL5RH2JUeEh3eoZxkJ2ZZJ9PVd1BcLioNuq4PRCZYxQ
    eosbarcelona  EOS8N1MhQpFQR3YABzVp4woPBywQnS5BunJtHv8jxtNQGrGEiTBhD
    eoscafeloons  EOS6XgXr6zGhyk6p2rSHrizUYCnhHCMjz3NuvBFGRATLUueBFC2tv
    eosdacserval  EOS5CJJEKDms9UTS7XBv8rb33BENRpnpSGsQkAe6bCfpjHHCKQTgH
    eosgenblockp  EOS8UAB1vDDKyLpWaFm3qQPXBFLEJxDcpqrjCTcaBXLguq5dPH1Bn
    eosnationftw  EOS8YhQPU6agWNfXYswnyM5fLaLecoriyD1H5Ckr3R95Z38eWZ9qg
    eosphereiobp  EOS7TTJ2LsMPbCPsRJSxL2PhhHBqg8xNfV3MtBcx7gzYxH27YGSHh
    eosriobrazil  EOS69gkEdoQsgY9enzWosFQhj2MPqWtHGyNhsiN1meG6msV4VBoqd
    evilproducer  EOS5iR3RpvpSM4Gm2EViLXUzsktHxjsAKRSUu8DZwJvVT1dTGwxfR
    greeneosiobp  EOS5uvrqhhwqpxVbRZRuxg2ahXRdVbzG9sDDc87QhY6oxKcQKCaKz
    ivote4eosusa  EOS8WHzxnaVoXek6mwU7BJiBbyugeqZfb2y2SKa7mVUv8atLfbcjK
    junglesweden  EOS79e4HpvQ1y1HdzRSqE1gCKhdN9kFGjjUU2nKG7xjiVCakt5WSs
    lioninjungle  EOS5g7gVXGeQVCTNW7U3gxeBJuvzvkmbCLioNqTUPYLjQXFGfQXfo
    ohtigertiger  EOS7tigERwXDRuHsok212UDToxFS1joUhAxzvDUhRof8NjuvwtoHX
    tokenika4tst  EOS6wkp1PpqQUgEA6UtgW21Zo3o1XcQeLXzcLLgKcPJhTz2aSF6fz

pending schedule empty

proposed schedule empty
```

**Comment by jrosich** [comment]
endpoint: 95.216.2.82:8080

**Comment by heifner** [comment]
The issue appears to be when nodeos is 2.0. The above endpoint is 1.8.4

```
07:11:13@heifnerk:~/ext/eosio/build$ programs/cleos/cleos --print-response -u http://95.216.2.82:8080 get schedule
RESPONSE:
---------------------
{
  "active": {
    "version": 276,
    "producers": [{
        "producer_name": "alohaeostest",
        "authority": [
          0,{
            "threshold": 1,
            "keys": [{
                "key": "EOS6CHhStoknSrJiQuKMezu54QsYUsWQNnooNiBfDUFzRQzfE9ruq",
                "weight": 1
              }
            ]
          }
        ]
      },{
        "producer_name": "atticlabjbpn",
        "authority": [
          0,{
            "threshold": 1,
            "keys": [{
                "key": "EOS5hVMcN6UVWtrNCxdp5HJwsz4USULmdfNA22UDyjRNdprXEiAP6",
                "weight": 1
              }
            ]
          }
        ]
      },{
```
VS:
```
07:12:31@heifnerk:~/ext/eosio/build$ programs/cleos/cleos --print-response -u https://jungle2.cryptolions.io:443 get schedule
RESPONSE:
---------------------
{
  "active": {
    "version": 276,
    "producers": [{
        "producer_name": "alohaeostest",
        "block_signing_key": "EOS6CHhStoknSrJiQuKMezu54QsYUsWQNnooNiBfDUFzRQzfE9ruq"
      },{
        "producer_name": "atticlabjbpn",
        "block_signing_key": "EOS5hVMcN6UVWtrNCxdp5HJwsz4USULmdfNA22UDyjRNdprXEiAP6"
      },{
```

**Comment by heifner** [comment]
Resolved by #8071, will be in `2.0.0-rc2`
